roles/openshift_node/tasks/systemd_units.yml

```diff
@@ -1,12 +1,19 @@
 ---
-- name: Install Node service file
-  template:
-    dest: "/etc/systemd/system/{{ openshift.common.service_type }}-node.service"
-    src: "{{ openshift.common.is_containerized | bool | ternary('openshift.docker.node.service', 'node.service.j2') }}"
-  when: not openshift.common.is_node_system_container | bool
-  notify:
-  - reload systemd units
-  - restart node
+- when: not openshift.common.is_node_system_container | bool
+  block:
+  - name: Install Node service file
+    template:
+      dest: "/etc/systemd/system/{{ openshift.common.service_type }}-node.service"
+      src: "{{ openshift.common.is_containerized | bool | ternary('openshift.docker.node.service', 'node.service.j2') }}"
+    notify:
+    - reload systemd units
+    - restart node
+    register: node_service
+
+  #TODO: systemd: daemon_reload=yes when we require ansible 2.4
```

Comment:
We're already using daemon_reload=yes elsewhere (docker role).  I believe this is working correctly on 2.3.x

Reply:
In 2.4 you can trigger only a reload and nothing else via the systemd module. Let me check to see if this is still necessary, this may have been a debugging artifact though.